fineract-provider/src/main/java/org/apache/fineract/portfolio/loanaccount/service/LoanWritePlatformServiceJpaRepositoryImpl.java

```diff
@@ -2727,6 +2727,8 @@ private CommandProcessingResult processLoanDisbursementDetail(final Loan loan, L
             }
         }
 
+        loan.updateLoanTransactionMappings();
```

Comment:
say, what about the code that does the copy for the transactions, maybe it is a better place to add this fix?

Reply:
I think regardless of where you use this method, it should still be a method of `Loan`.

Reply:
true, but I asked if ut should be called somewhere else...